### PR TITLE
[Minor] Removed unused variables and parameters

### PR DESCRIPTION
**Pull request by derrabus**

### CODE REVIEW

File: src/Tests/Violation/ViolationChecker/InterfaceViolationCheckerTest.php [hunk 1]
```diff
@@ -32,7 +32,7 @@ public function testCheck()
         $deprecatedInterfaceUsage = $deprecatedInterfaceUsage->reveal();
 
         $ruleSet = $this->prophesize('SensioLabs\DeprecationDetector\RuleSet\RuleSet');
-        $file = $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
+        $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
```
Comment (slde-gorillaman) [inline src/Tests/Violation/ViolationChecker/InterfaceViolationCheckerTest.php]: This line could be remove completely, if $file is not used anywhere in the test, then it makes no sense creating the mock.

File: src/Tests/Violation/ViolationChecker/SuperTypeViolationCheckerTest.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ public function testCheck()
         $deprecatedSuperTypeUsage = $deprecatedSuperTypeUsage->reveal();
 
         $ruleSet = $this->prophesize('SensioLabs\DeprecationDetector\RuleSet\RuleSet');
-        $file = $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
+        $this->prophesize('Symfony\Component\Finder\SplFileInfo')->reveal();
```
Comment (slde-gorillaman) [inline src/Tests/Violation/ViolationChecker/SuperTypeViolationCheckerTest.php]: Same as in InterfaceViolationCheckerTest, this line can be removed completely

File: src/Tests/Violation/ViolationChecker/InterfaceViolationCheckerTest.php (hunk 1 above)
Comment (derrabus) [inline src/Tests/Violation/ViolationChecker/InterfaceViolationCheckerTest.php]: Right! X-)

**Comment by derrabus** [comment]
I've updated the PR according to @slde-gorillaman's comments and rebased against master to resolve conflicts.

**Pull request merged**